Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+ruby '2.4.0'
```

Comment:
Fixing ruby version in Gemfile can cause issues with deployment to ElasticBeanstalk, I think. We can try to do it, but I think `.ruby-version` file is better place for this.